packages/types/src/networks.ts

```diff
@@ -30,6 +30,9 @@ export enum NetworkNames {
   Edgeware = "EDG",
   EdgeEVM = "evmEDG",
   ZkSyncGoerli = "zkSyncGoerli",
+  Unique = "UNQ",
+  Quartz = "QTZ",
+  Opal = "OPL",
```

Comment:
since you are on both evm and substrate, you have to create unique names, otherwise itll conflict with each other

Reply:
Done